### PR TITLE
Feat:Added Landing Page

### PR DESCRIPTION
**Pull request by neetukumari4858**
Feat:Added Landing Page

Screenshot from 2022-03-26 15-44-15
Screenshot from 2022-03-26 15-46-04

### CODE REVIEW

File: src/Components/Home-page-Card/Data.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        subtitle:"upto 20% off"
+    },
+    {
+        image:"/image/shoe5.jpeg",
+        subtitle:"upto 15% off"
+    },
+]
+const Data_Two=[
+    {
+        image:"/image/shoe13.webp",
```
Comment (mariyasada) [inline src/Components/Home-page-Card/Data.js]: suggested that use proper name like if you have top brand search then use a name like topBrandData

File: src/Components/Navigation/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+//             <li><Link className='nav-link icon-content' to ="/">  Home</Link></li>      
+//             <li><Link className='nav-link icon-content' to ="/ProductPage">  Shop</Link></li>
+//             <li><Link className='nav-link icon-content' to ="/ProductPage">  Men</Link></li>
+//             <li><Link className='nav-link icon-content' to ="/Product">  Woman</Link></li>
+//         </ul>
+//     </div>
+//     <div className="nav-section">
+//         <div className="outer-search-bar-div">
+
+//             <input type="text" className="search-bar" placeholder="   Search.."></input>
```
Comment (mariyasada) [inline src/Components/Navigation/Navbar.js]: delete commented code

File: src/Pages/HomePage/HomePage.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { Footer } from "../../Components/Footer/Footer";
+import { Cards } from "../../Components/Home-page-Card/CardSection";
+import { Navbar} from "../../Components/Navigation/Navbar";
+import "./HomePage.css";
```
Comment (mariyasada) [inline src/Pages/HomePage/HomePage.js:4]: export all components in index.js one file and import from that particular file

**Review by mariyasada — APPROVED** [review]
I have added some suggestion just look at once

**Comment by ckahub** [comment]
suggestion: use link instead of putting all images in current repository, website will load fater

**Comment by neetukumari4858** [comment]
> suggestion: use link instead of putting all images in current repository, website will load f

**Pull request merged**